docs/components/query.md

```diff
@@ -560,14 +560,32 @@ Flags:
 
 ```
 
-## Exported metrics
-
-Thanos Query also exports metrics about its own performance. You can find a list with these metrics below.
-
-**Disclaimer**: this list is incomplete. The remaining metrics will be added over time.
-
-| Name                                    | Type      | Labels                                          | Description                                                                                                       |
-|-----------------------------------------|-----------|-------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
-| grpc_client_handled_total               | Counter   | grpc_code, grpc_method, grpc_service, grpc_type | Number of gRPC client requests handled by this query instance (including errors)                                  |
-| grpc_server_handled_total               | Counter   | grpc_code, grpc_method, grpc_service, grpc_type | Number of gRPC server requests handled by this query instance (including errors)                                  |
-| thanos_store_api_query_duration_seconds | Histogram | samples_le, series_le                           | Duration of the Thanos Store API select phase for a query according to the amount of samples and series selected. |
```

Comment:
What about these 3 metrics here? Why are they being removed?

Reply:
The `grpc` metrics were an oversight on my part, I have reinstated both metrics.  
The `thanos_store_api_query_duration_seconds` is a Thanos Store metric which is reflected in the Thanos Store metrics documentation.

Reply:
@TomiwaAribisala-git `thanos_store_api_query_duration_seconds` is not a metrics from the Thanos Store Gateway component. It's a metric from the Thanos Store API, which is the standard interface for data query in Thanos. All the Thanos components that can b "queried" implement this API (like Ruler, Receive, Query, Store Gateway). 

So it is a metric of all those components I mentioned.

The confusion between Store API and Store Gateway is common.

Reply:
@douglascamata, thanks for clarifying. 
There are some metrics that were not Thanos "components" metrics I noticed when I started documenting, given your clarification I consider `thanos_store_api_query_duration_seconds` to be one of them; 

I intend to document these metrics as Thanos "General Metrics" which are metrics that are not specifically for one Thanos component but cuts across several components, I can do this by writing the General Metrics in a `metrics.md` file in the `thanos/docs` directory, additionally I will write the `thanos_store_api_query_duration_seconds` to the `metrics.md` file and remove it from the Thanos Store Gateway list of metrics. 

I await your opinion on this.

Reply:
Sounds good to me! 👍

Reply:
All Thanos components metrics and Thanos General metrics are done.